src/lib/markbind/src/lib/markdown-it-shared/markdown-it-emoji-fixed.js

```diff
@@ -1,15 +1,13 @@
 // fix emoji numbers
 const emojiData = require('markdown-it-emoji/lib/data/full.json');
 // Extend emoji here
-emojiData['zero'] = emojiData['0'] = '<img style="height: 1em;width: 1em;margin: 0 .05em 0 .1em;vertical-align: -0.1em;" src="https://assets-cdn.github.com/images/icons/emoji/unicode/0030-20e3.png">';
```

Comment:
It seems like we are also populating the digit key here (i.e. `emojiData['1']`) alongside the letter key, but we don't seem to do that in the new version. Is that intentional?

Reply:
Also I strongly encourage adopting the existing code's format (i.e. assigning each individual emoji line by line). The new changes disassociates the mapping between the names and the corresponding emoji.

Reply:
> It seems like we are also populating the digit key here (i.e. `emojiData['1']`) alongside the letter key, but we don't seem to do that in the new version. Is that intentional?

I did considering keeping the digit mapping for backwards compatibility. However, as they aren't standard [emoji](https://www.webfx.com/tools/emoji-cheat-sheet/), and I could not find any current uses in the SE textbook and Markbind docs, I thought it would be better to remove them.

---

> Also I strongly encourage adopting the existing code's format (i.e. assigning each individual emoji line by line). The new changes disassociates the mapping between the names and the corresponding emoji.

OK, I see your point. I will update the PR.